sidenav.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sidenav</title>
+    <style>
+        html {
+            border-left: 5px solid rgb(240, 186, 186); margin: 0; padding: 0; height: 350px;
```

Comment:
Please move each css property per line. It's not readable to put all in one line.